Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -29,58 +45,42 @@ Please keep only your sitepackage extension or any other extension, which was
 explicitly built for your current project and does not have an own Git
 repository.
 
-Configure composer
+Configure Composer
 ==================
 
-Create a file with name :file:`composer.json` in your project root, not inside
-your web root.
+Create a file with name :ref:`composer.json <t3coreapi:files-composer-json>`
+in your project root, not inside your web root.
 
-You can use the :file:`composer.json` from typo3/cms-base-distribution as an
+You can use the :file:`composer.json` from `typo3/cms-base-distribution` as an
 example. Use the file from the branch which matches your current version, for
 example `12.x <https://github.com/typo3/TYPO3.CMS.BaseDistribution/tree/12.x/composer.json>`__.
 
 However, this may require extensions you don't need or omit extensions you do
 need, so be sure to update the required extensions as described in the next
 sections.
 
-The :file:`composer.json` in the Base distribution includes a scripts section:
-
-.. code-block:: json
-   :caption: /composer.json
-
-   {
-      "scripts": {
-         "typo3-cms-scripts": [
-            "typo3cms install:generatepackagestates",
-            "typo3cms install:fixfolderstructure"
-         ],
-         "post-autoload-dump": [
-            "@typo3-cms-scripts"
-         ]
-      }
-   }
-
-
-This requires helhum/typo3-console (so be sure to require that too) and is essential
-for generating the file :file:`typo3conf/PackageStates.php`.
-
+You can also create the :file:`composer.json` file via a :bash:`composer init` command.
+Or use the `TYPO3 Composer Helper <https://get.typo3.org/misc/composer/helper>`__.
+Also advanced project builders like `CPS-IT project-builder <https://github.com/CPS-IT/project-builder>`__
+help you to easily initialize this most vital file of a Composer project.
 
 .. hint::
 
-   The script section may not be available in current `main` branch of the Base Distribution.
-   Look at the previous versions.
+   If you see versions of the :file:`composer.json` for versions older than TYPO3 v12,
```

Comment:
This sentence is hard to read. What do you want to say?

Reply:
When you migrate a project from lgacy you don't have a composer.json file, and therefore no scripts section. Or do I miss something?

Reply:
That is true, but people (at least people I support) tend to frankenstein their composer.jsons) or might have the old version of this documentation. So they might see diffetent composer.jeons, and I think it would help to clear up why the old suggestion no longer is needed here.

It's not a hill I'm willing to die on, so if you say to remove it, I'll play along 😅

Reply:
I don't get it: So, you see legacy installations with a project-level composer.json, which is obviously not in use?

Reply:
If I google and look at documentation and snippets in the internet, this is where I see diferrent composer.json approaches, and this is what I want to educate people on, why they might see things that they see.